src/modules/user/routes.js

```diff
@@ -48,3 +50,9 @@ usersRouter.delete(
   validateUserPermissions,
   handleDelete
 );
+
+//post /users/resetPassword
+usersRouter.get("/resetPassword", sendResetPassword)
+
+//post /users/newPassword
+usersRouter.get("/newPassword:resetToken",)
```

Comment:
Think this should be like so
```suggestion
usersRouter.get("/newPassword/:resetToken",)
```